src/snapshot.c

```diff
@@ -87,6 +89,12 @@ struct snapshot *get_snapshot(int pid, int *tids, int *index, int nr_tids)
         goto get_snapshot_fail;
     }
 
+    /*
+     * get thread states if requested by user
```

Comment:
Don't think this comment is useful, the code is clear.

Reply:
Removed this comment: https://github.com/tbricks/tbstack/pull/14/commits/bbfa5131d6570126f1c5dce045c485340634e149